Audiobook Player/Player/Containers/PlayerJumpIconView.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  PlayerJumpIconView.swift
+//  Audiobook Player
+//
+//  Created by Florian Pichler on 22.04.18.
+//  Copyright © 2018 Tortuga Power. All rights reserved.
+//
+
+import UIKit
+
+enum PlayerJumpIconType {
```

Comment:
Is this being used somewhere?

Reply:
Was replaced by the two subclasses below, will remove.